### DISCUSSION TITLE
How to ignore styling?

### DISCUSSION THREAD

**Discussion by ghanteyyy** [TomSchimansky/CustomTkinter#861] [General]
I want to ignore styling. How do I do it?

**Comment by TomSchimansky** [comment]
?

**Comment by ghanteyyy** [comment]
If I don't want to add styling in a specific widget then how can I do it ? I think I have once added a **parameter** and set it to **False** then no styling was added to that widget.

**Comment by Akascape** [comment]
@ghanteyyy what exactly you mean by styling? 

Customtkinter is not a style or theme for tkinter. It is a total different set of widgets build with canvas of tkinter. So styling will not work with them.
But you can use normal tkinter widgets with customtkinter. 
Those tkinter widgets can be styled with tkinter themes but it will have no impact on the ctk ones.

Customtkinter only has color themes which is not a style.
